### PR TITLE
move source files into src folder

### PR DESCRIPTION
**Pull request by donkey-donkey**

### CODE REVIEW

File: .idea/.gitignore [hunk 1]
```diff
@@ -0,0 +1,8 @@
+# Default ignored files
```
Comment (aswever) [inline .idea/.gitignore:1]: .idea should be in your global .gitignore and removed from the PR

**Review by aswever — DISMISSED** [review]
remove `.idea/`, otherwise i approve of this if @the-frey does

**Comment by the-frey** [comment]
Apart from the structure issue, LGTM

**Comment by donkey-donkey** [comment]
i ran prettier but it still says build failing.

**Review by the-frey — APPROVED** [review]

**Pull request merged**